src/components/ComponentPages/TopicDetails/CampTreeCard/index.tsx

```diff
@@ -28,38 +30,40 @@ const CampTreeCard = ({ scrollToCampStatement, getSelectedNode }) => {
     subscribeToCampApi(reqBody);
   }
   return (
-    <Card
-      className={"ctCard canCard mb-3 " + styles.ctCard}
-      title={<h3>Canonizer Sorted Camp Tree</h3>}
-      extra={
-        <>
+    <Collapse defaultActiveKey={['1']} expandIconPosition="right" className="topicDetailsCollapse">
+      <Panel header={<h3>Canonizer Sorted Camp Tree</h3>} 
+        key="1" extra={
+          <>
+          <div className="ant-checkbox-wrapper" onClick={event => {event.stopPropagation();}}>
           <Checkbox
             checked={
               currentCampRecord && currentCampRecord[0].campSubscriptionId
```

Comment:
This probably replaced as `!!(currentCampRecord && currentCampRecord[0].campSubscriptionId)`

Reply:
we don't need checkbox here, so it will be removed in next  PR